src/DataLayer/helpers/filters.ts

```diff
@@ -1,29 +1,56 @@
 // (C) 2007-2018 GoodData Corporation
 import cloneDeep = require('lodash/cloneDeep');
-import isEmpty = require('lodash/isEmpty');
 import { AFM } from '@gooddata/typings';
+import isDateFilter = AFM.isDateFilter;
+import isNegativeAttributeFilter = AFM.isNegativeAttributeFilter;
+import isPositiveAttributeFilter = AFM.isPositiveAttributeFilter;
 
-function isDateFilter(filter: AFM.CompatibilityFilter): filter is AFM.DateFilterItem {
-    return !!(
-        (filter as AFM.IAbsoluteDateFilter).absoluteDateFilter
-        || (filter as AFM.IRelativeDateFilter).relativeDateFilter
-    );
+/**
+ * Tests whether attribute elements are empty of not.
+ *
+ * @param elements one of the acceptable attribute element forms
+ */
+export function isEmptyAttributeElements(elements: string[]): boolean {
```

Comment:
could be this simplified by using `lodash/isEmpty`?

Reply:
changed body to use isEmpty(); i prefer to keep the isEmptyAttributeElements() - have the naming 'domain' specific (+ in the future if/when we migrate changes from ExecuteAFM to AFM, the attribute elements can be of 2-3 different types => we can extend this method)